### PR TITLE
test(frontend): Missing tests for exchange derived

### PR DESCRIPTION
**Pull request by AntonioVentilii**
# Motivation

Some missing hedge cases for the exchange derived, to raise our coverage.

### CODE REVIEW

File: src/frontend/src/tests/lib/derived/exchange.derived.spec.ts [hunk 1]
```diff
@@ -479,6 +479,76 @@ describe('exchange.derived', () => {
...
+			};
+
+			icrcDefaultTokensStore.set({ data: mockUnknownToken, certified: false });
+			icrcCustomTokensStore.setAll([
+				{ data: { ...mockUnknownToken, enabled: true }, certified: false }
+			]);
+
+			exchangeStore.set([{ ethereum: ethPrice }]);
+
+			expect(get(exchanges)?.[mockUnknownToken.id]).toBeUndefined();
```
[the remark this answers is not in the document]
Comment (AntonioVentilii) [reply src/frontend/src/tests/lib/derived/exchange.derived.spec.ts]: done

File: src/frontend/src/tests/lib/derived/exchange.derived.spec.ts [hunk 2]
```diff
@@ -479,6 +479,76 @@ describe('exchange.derived', () => {
 			});
 		});
 
+		it('should fallback to solana price for ICRC tokens with solana exchangeCoinId', () => {
+			const mockSolToken: IcCkToken = {
+				...mockValidIcCkToken,
+				id: parseTokenId('CkSolTokenId'),
+				exchangeCoinId: 'solana' as const
+			};
```
[the remark this answers is not in the document]
Comment (AntonioVentilii) [reply src/frontend/src/tests/lib/derived/exchange.derived.spec.ts:488]: done
